apps/discussions/package.json

```diff
@@ -50,10 +52,10 @@
     "start:app": "cd app && npm start && cd ..",
     "test": "cross-env TRUFFLE_TEST=true npm run ganache-cli:test",
     "compile": "aragon contracts compile",
-    "build": "npm run sync-assets && npm run build:app && npm run build:script",
+    "build": "npm run compile && npm run sync-assets && npm run build:app && npm run build:script",
```

Comment:
Compile step is probably not needed here, as it is part of the regular build during `aragon apm publish` so it would eventually run twice